samples/basic/blinky/boards/pocketbeagle_2_am6232_m4.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		led0 = &led0;
```

Comment:
Is this proper regarding tabs/spaces?

Reply:
I am using tabs, which I think is correct. And there are no errors in CI.